specification/devcenter/data-plane/Microsoft.DevCenter/preview/2023-07-01-preview/environments.json

```diff
@@ -1211,6 +1211,38 @@
         ]
       }
     },
+    "EnvironmentOperationStatus": {
```

Comment:
This makes me wonder, though: why isn't the `EnvironmentOperation` an LRO: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs ? Did we discuss this in review? I don't recall.

In that case, we have rigid requirements for LROs to make sure generated code works correctly. See the expected status monitor enum here: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#lro-status-monitor-structure

Reply:
Apologies, think we didn't make this especially clear -- any operation that shows up in the Dev Box operations/Environment operations APIs will be an LRO and will have an associated status monitor with the OperationStatus model. We're using a different model here to avoid any breaking changes with the current flow, and because we will have different properties on these resource-specific operations which may not be applicable to other operations (such as `environmentParameters`). I think aligning the enum would make sense though, I've added the `NotStarted` value to these.